Recipes/Recipes.API/appsettings.json

```diff
@@ -4,6 +4,9 @@
     "Issuer": "https://identity.recipesproject.com",
     "Audience": "https://identity.recipesproject.com"
   },
+  "TagCreation": {
```

Comment:
Perhaps `AllowImplicitTagCreation`

Reply:
```TagCreation``` is the name of a section. I agree that ```TagCreation``` is probs not the best section name, but ```AllowImplicitTagCreation``` seems more like an option name, not an option group name.

Reply:
Changed section to FeatureToggles and made option name AllowImplicitTagCreation